### PR TITLE
Added mysql support

### PR DESCRIPTION
**Pull request by arealdeadone**
### Added Mysql support

The user can now automatically backup their Mysql database using the file. 
Basically created a function 'mysqldump' which executes the shell command:

``` sh
    mysqldump -uUSERNAME -pPASSWORD -hHOSTNAME DATABASE_NAME > DEST_DIR/Filename
```

and upload it to S3 using the existing code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,28 +25,43 @@ The file should have the following format:
...
+          "key": "your_s3_key",
+          "secret": "your_s3_secret",
+          "bucket": "s3_bucket_to_upload_to",
+          "destination": "/mysql",
+          "encrypt": true
         }
       },
       "cron": {
-        "time": "11:59",
+        "crontab": "* * * * * *"
```
Comment (MrDHat) [inline README.md:60]: Why was this change needed?

**Comment by MrDHat** [comment]
Please remove the test file and then, squash and force push so that your password doesn't get logged

File: README.md (hunk 1 above)
Comment (arealdeadone) [reply README.md:60]: Actually I did not change that. I guess I pulled the code from the other branch and sent a PR for the master branch
